### PR TITLE
docs-cn: make chunk rpc docs easy to understand.

### PR DESCRIPTION
**Pull request by wshwsh12**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v4.0 changes for now)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: reference/configuration/tidb-server/configuration-file.md [hunk 1]
```diff
@@ -378,7 +378,7 @@ prepare 语句的 Plan cache 设置。
 
 ### `enable-chunk-rpc`
```
Comment (SunRunAway) [inline reference/configuration/tidb-server/configuration-file.md]: Is this option worth being public? It breaks `Don't Make Me Think` law. 
Users would touch this option only when they met the occurrence of bug. IMO, this option could stay in the environment variable and be retired sometime.

Comment (SunRunAway) [reply reference/configuration/tidb-server/configuration-file.md]: @zz-jason PTAL, too.

File: reference/configuration/tidb-server/configuration-file.md [hunk 2]
```diff
@@ -378,7 +378,7 @@ prepare 语句的 Plan cache 设置。
...
 
-+ 开启 coprocessor 的 `Chunk` 数据编码格式。
++ 开启 coprocessor 的 `Chunk` 数据编码格式。可通过 PD 动态配置该选项，且对该实例新建的 session 生效。
```
Comment (TomShawn) [inline reference/configuration/tidb-server/configuration-file.md]:
```suggestion
+ 开启 Coprocessor 的 `Chunk` 数据编码格式。可通过 PD 动态配置该选项，且对该实例新建的 session 生效。
```

File: reference/configuration/tidb-server/tidb-specific-variables.md [hunk 3]
```diff
@@ -666,3 +666,11 @@ TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在
...
 
 这个变量用来控制是否开启 statement summary 功能。如果开启，SQL 的耗时等执行信息将被记录到系统表 `performance_schema.events_statements_summary_by_digest` 中，用于定位和排查 SQL 性能问题。
+
+### tidb_enable_chunk_rpc <span class="version-mark">从 v4.0 版本开始引入</span>
+
+作用域：SESSION
+
+默认值：1
+
+这个变量用来设置是否启用 coprocessor 的 `Chunk` 数据编码格式。
```
Comment (TomShawn) [inline reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
这个变量用来设置是否启用 Coprocessor 的 `Chunk` 数据编码格式。
```

File: reference/configuration/tidb-server/configuration-file.md (hunk 1 above)
Comment (zz-jason) [reply reference/configuration/tidb-server/configuration-file.md]: agree

**Comment by TomShawn** [comment]
@wshwsh12 Please address the above comments, thanks！

**Pull request merged**